Lib.AspNetCore.ServerTiming/Processors/RemoveAllMetricsProcessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Lib.AspNetCore.ServerTiming.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+
+namespace Lib.AspNetCore.ServerTiming.Processors
+{
+    /// <summary>
+    /// A processor to clear all metrics in the response
+    /// </summary>
+    public class RemoveAllMetricsProcessor : IServerTimingProcessor
```

Comment:
I don't think there is a value behind having this one (I believe you are also skipping this one in docs).